pom.xml

```diff
@@ -15,8 +26,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.enforced.version>[17,18)</java.enforced.version>
         <maven.compiler.release>11</maven.compiler.release>
-
-        <rosetta.dsl.version>9.4.0</rosetta.dsl.version>
+        <repoServerHost>s01.oss.sonatype.org</repoServerHost>
+        <rosetta.dsl.version>0.0.0.projection-tabulation-SNAPSHOT</rosetta.dsl.version>
```

Comment:
Reminder for us to release DSL and set this version accordingly when everything is working.